resource/reactivedrop_brazilian.txt

```diff
@@ -2116,6 +2116,7 @@
 "asw_speech_ready"		"Prontos?"
 "[english]asw_speech_smile"		"*smile*"
 "asw_speech_smile"		"*sorriso*"
+// Your character will say an random idle-chatter line. Here's a list of possible outcomes https://github.com/ReactiveDrop/reactivedrop_public_src/blob/reactivedrop_beta/src/game/server/swarm/asw_marine_speech.cpp#L873
```

Comment:
if you're linking to a line of code somewhere permanent like this, it should use the commit hash rather than the branch name in case the code above that line gets longer or shorter in the future and the line number changes - click the three dots and then "copy permalink".

Reply:
Done